README.md

```diff
@@ -187,6 +187,67 @@ result = my_zosmf.get_info()
 ```
 The result will be a JSON object containing z/OSMF information
 
+## Local setup for macOs
+
+Make sure that the current python version is `3.10` or older
+
+If it is in an older version , you can downgrade to a lower versions by following the below steps:
+
+```bash
+brew install pyenv
+
+echo 'PATH=$(pyenv root)/shims:$PATH' >> ~/.zshrc
+
+pyenv install 3.10 ## Example for python 3.10 version
```

Comment:
`brew` may not be installed for some users, but it isn't listed as a pre-requisite for the added steps.